packages/core/src/item.ts

```diff
@@ -278,6 +278,28 @@ export interface AuditResult {
     fieldIndex: number;
 }
 
+export class ItemHistory extends Serializable {
+    constructor(vals: Partial<ItemHistory> = {}) {
+        super();
+        Object.assign(this, vals);
+    }
+
+    date: Date = new Date();
+
+    updatedBy: AccountID = "";
+
+    name?: string;
+
+    vaultId?: string;
+
+    @AsSerializable(Field)
+    fields?: Field[];
```

Comment:
FYI, properties with a serialisation decorator such as `@AsSerializable`, `@AsDate` are an exception and don't necessarily need to be initialized.